### PR TITLE
Feature/update placeholder text

### PR DESCRIPTION
**Pull request by comountainclimber**
**What current issue(s) from Trello/Github does this address?**
There is no issue currently open but @drptbl noticed that our placeholders dont match the mocks and I agree they are verbose...This updates placeholders to consistently just use`Password`.

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

File: app/containers/LoginNep2/LoginNep2.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default class LoginNep2 extends Component<Props, State> {
             />
           </div>
           <PasswordInput
-            placeholder="Enter your passphrase here"
+            placeholder="Password"
```
Comment (mhuggins) [inline app/containers/LoginNep2/LoginNep2.jsx]: Up on line 34 is a verbose placeholder "Enter your encrypted key here".  Should we update that as well, perhaps to say "Encrypted Key"?

Comment (comountainclimber) [reply app/containers/LoginNep2/LoginNep2.jsx]: 👍 sounds good @mhuggins

**Comment by mhuggins** [comment]
@comountainclimber This broke a test, you'll need to update `LoginNep2.test.js` to change "Enter your encrypted key here" to "Encrypted Key".

File: app/containers/LoginNep2/LoginNep2.jsx [hunk 2]
```diff
@@ -31,15 +31,15 @@ export default class LoginNep2 extends Component<Props, State> {
         <form onSubmit={this.handleSubmit}>
```
Comment (affendy93) [inline app/containers/LoginNep2/LoginNep2.jsx]: 😁

Comment (comountainclimber) [reply app/containers/LoginNep2/LoginNep2.jsx]: ???

**Pull request merged**